samples/client/petstore/ruby/Gemfile.lock

```diff
@@ -13,7 +13,7 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
     autotest (4.4.6)
       ZenTest (>= 4.4.1)
-    autotest-fsevent (0.2.13)
+    autotest-fsevent (0.2.14)
```

Comment:
updated from 0.2.13 to 0.2.14 because was getting doing a `bundle install` with 0.2.13 was getting:
```
Darwin 18 is not (yet) supported (RuntimeError)
```
upgrading to 0.2.14 fixed issue.